### PR TITLE
give a useful error message when Qt5X11Extras is not installed

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

File: build/depends.py [hunk 1]
```diff
@@ -310,6 +308,10 @@ def configure(self, build, conf):
             elif not qt5 and not conf.CheckForPKG('QtCore', '4.6'):
                 raise Exception('QT >= 4.6 not found')
 
+            if not conf.CheckLib('Qt5X11Extras'):
```
Comment (daschuer) [inline build/depends.py]: The qt5 guard is gone.

File: build/depends.py [hunk 2]
```diff
@@ -310,6 +308,10 @@ def configure(self, build, conf):
...
+                raise Exception('Could not find Qt5X11Extras or its development headers')
+            build.env.Append(LIBS = 'Qt5X11Extras')
```
Comment (daschuer) [inline build/depends.py]: I think this is redundant.

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you for taking care.

File: build/depends.py (hunk 1 above)
Comment (Be-ing) [reply build/depends.py]: So? Qt4 is not supported.

Comment (daschuer) [reply build/depends.py]: Ah, yes right.

File: build/depends.py (hunk 2 above)
Comment (Be-ing) [reply build/depends.py]: You're right. It works when this line is removed.

**Pull request merged**